content/client-server-api/modules/receipts.md

```diff
@@ -99,7 +212,7 @@ format of the EDUs are:
 {
     <room_id>: {
         <receipt_type>: {
-            <user_id>: { <content> }
+            <user_id>: { <content (ts & thread_id, currently)> }
```

Comment:
I suspect there should be corresponding updates to the [server-server API for receipts](https://pr1255--matrix-spec-previews.netlify.app/server-server-api/#mreceipt), at least to "Read Receipt Metadata".